src/inet/InetLayer.cpp

```diff
@@ -31,9 +31,11 @@
  *        * UDP network transport
  *        * Raw network transport
  *
- *      For BSD/POSIX Sockets, event readiness notification is handled
- *      via file descriptors and a traditional poll / select
- *      implementation on the platform adaptation.
+ *      For BSD/POSIX Sockets (CHIP_SYSTEM_CONFIG_USE_LWIP), event readiness
```

Comment:
```suggestion
 *      For BSD/POSIX Sockets (CHIP_SYSTEM_CONFIG_USE_SOCKETS), event readiness
```

Reply:
Yes. I will definitely fix and add substantially more documentation.